Tests/FluentAssertions.Specs/Specialized/TaskAssertionSpecs.cs

```diff
@@ -143,5 +143,21 @@ async Task CheckContextAsync()
                 SynchronizationContext.Current.Should().NotBeNull();
             }
         }
+
+        [UIFact]
```

Comment:
🤔 This test belongs in the `CallerIdentifierSpecs`
🔧 I think rephrasing this test to `Caller_identification_should_also_work_for_statements_following_async_code` will make the purpose of the test much clearer.

Reply:
Done in https://github.com/fluentassertions/fluentassertions/pull/1770/commits/cde578c4ba0b3a0b7d1ccea9972ec4f6aa2b366d.